packages/vue-flicking/README.md

```diff
@@ -0,0 +1,126 @@
+# @egjs/vue-flicking [![version](https://img.shields.io/npm/v/@egjs/vue-flicking.svg?style=flat)](https://www.npmjs.com/package/@egjs/vue-flicking)
+> 🎠 A module used to implement flicking interactions. With this module, you can make flicking gestures, which are ways to navigate left and right to move between panels arranged side by side.
+
+Vue wrapper of [@egjs/flicking](https://github.com/naver/egjs-flicking)
+
+## Install
+```sh
+npm install --save @egjs/vue-flicking
+```
+
+## ❗Changes from @egjs/flicking
+- All `camelCased` event names became **`kebab-case`**
+  - e.g., `moveEnd` => **`move-end`**
+- You can't use methods that manipulates DOM directly
+  - e.g., `append()`, `remove()`, ...
+
+## Quick Start
+### Global registration
+```js
+import VueFlicking from "@egjs/vue-flicking";
+Vue.use(VueFlicking);
+```
+
+### Local registration
+```js
+import { Flicking } from "@egjs/vue-flicking";
+
+export default {
+  components: {
+    Flicking: Flicking,
+  }
+}
+```
+
+### Usage
+```vue
+<template>
+  <flicking
+    :options="{ gap: 10, moveType='freeScroll' }"
+    :tag="'div'"
+    :plugins="plugins"
+    @need-panel="e => {
+      // DO SOMETHING
```

Comment:
@WoodNeck It would be useful to comment more helpful information. like follows
```js
// ADD PANELS
```

Reply:
Sure, why not.
I'll fix both comments.